sematic/resolvers/local_resolver.py

```diff
@@ -222,6 +245,26 @@ def _cancel(data):
         self._create_resolution(self._root_future)
         self._update_resolution_status(ResolutionStatus.RUNNING)
 
+        self._cache_namespace = self._resolve_cache_namespace()
+
+    def _resolve_cache_namespace(self) -> Optional[str]:
+        if self._raw_cache_namespace is None:
+            logger.info("The Resolution is not configured to use cached values")
```

Comment:
This message is not useful. We don't want to log when things are NOT happening, especially when it is the most frequent. Proposition:
- change this to `.debug`
- when encountering a `cache=True` function, do log whether the function will use a cache (both yes and no)